src/backend/joanie/core/filters/admin/__init__.py

```diff
@@ -276,6 +276,10 @@ class Meta:
     )
     state = filters.ChoiceFilter(choices=enums.ORDER_STATE_CHOICES)
     ids = MultipleValueFilter(field_class=fields.UUIDField, field_name="id")
+    created_on = filters.DateFilter(field_name="created_on", lookup_expr="exact")
+    created_on_after = filters.DateFilter(field_name="created_on", lookup_expr="gte")
+    created_on_before = filters.DateFilter(field_name="created_on", lookup_expr="lte")
+    created_on_date_range = filters.DateFromToRangeFilter(field_name="created_on")
```

Comment:
So according to the discussion of this morning, did you check if the `DateFromToRangeFilter` is not enough ?

Reply:
Yes, I can confirm you that the `DateFromToRangeFilter` is enough to filter our queries the way we want.
We can either use both filter arguments make a range, or only one to satisfy our filtering if we want dates prior to the given date, or later ones.

For e.g, by only using one out of two parameters : 
- `created_on_date_range_after` in the query filter, it's the same behavior than  `created_on_after` and gives same results.
- `created_on_date_range_before` in the query filter, it's the same behavior than `created_on_before` and gives same results.

Reply:
So, you can remove useless filters

Reply:
Sorry, I thought I removed them already.